### PR TITLE
Widget: new signals

### PR DESCRIPTION
**Pull request by jerneju**
##### Description of changes
- [x] Base Vectorizer (Bag of Words, Similarity Hashing)
- [x] Concordance
- [x] Corpus Viewer
- [x] Word Cloud
- [x] Corpus (Load Corpus)
- [x] The Guardian
- [x] NY Times
- [x] Twitter
- [x] Wikipedia
- [x] Preprocess Text
- [x] Duplicate Detection
- [x] Tweet Profiler
- [x] Topic Modelling
- [x] Word Enrichment
- [x] GeoMap

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by nikicc** [comment]
Once this is done, please don't forget to bump the Orange version in the requirements to the version in which these new signals have been introduced. Currently we have `Orange3>=3.3.10`, which, I believe, didn't have these new signals yet.

**Comment by jerneju** [comment]
Thanks. New signals were introduced in version `3.4.3`.

**Comment by nikicc** [comment]
Could you please rebase? OWWordCloud was updated meanwhile 🙈

**Comment by jerneju** [comment]
Rebased.

File: orangecontrib/text/widgets/owconcordance.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 from Orange.widgets import gui
 from Orange.widgets.settings import Setting, ContextSetting, PerfectDomainContextHandler
-from Orange.widgets.widget import OWWidget, Msg
+from Orange.widgets.widget import OWWidget, Msg, Input,Output
```
Comment (nikicc) [inline orangecontrib/text/widgets/owconcordance.py]: Could you add a space after the comma?

Comment (jerneju) [inline orangecontrib/text/widgets/owconcordance.py]: Fixed.

**Pull request merged**